### DISCUSSION TITLE
[Support]: How do you improve your Score History in 0.14?

### DISCUSSION THREAD

**Discussion by rwlove** [blakeblackshear/frigate#12339] [General Support] [answered]
### Describe the problem you are having

It's hard for me to remember the pre-0.14 UI exactly, so bear with me stumbling through the terminology.

If I understood pre-0.14 correctly, if you marked a detection as false (or was it by deleting?) it would improve the score history and thus improve future object detections.

How is this accomplished in 0.14?

I understand Frigate Plus (http://frigate/plus) will better train the model, but can you no longer improve detection by ensuring you have a good score history?

### Version

0.14.0

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
N/A
```

### FFprobe output from your camera

```shell
N/A
```

### Frigate stats

```json
N/A
```

### Operating system

Other Linux

### Install method

Docker CLI

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

N/A

**Comment by NickM-27** [comment, marked as answer]
the frigate+ page can still be used for this. To be clear it was never that impactful in terms of improving object detection results. It only applies to the initial detection when motion is detected in an area that an object is not currently known to be in, and marking a false positive just excluded that item from the data that is used. All results are included by default so especially over time the impact of a false positive being included in that data is negligible
